### ISSUE TITLE
doc says to login as a developer, and then immediatly shows a command ran as an admin

### ISSUE THREAD

**Issue by cfergeau** [code-ready/crc#2176]
https://github.com/code-ready/crc/blob/master/docs/source/topics/proc_accessing-the-openshift-cluster-with-oc.adoc

> 3. Log in as the developer user:
> [...]
> 4. You can now use oc to interact with your OpenShift cluster. For example, to verify that the OpenShift cluster Operators are available, you need to be logged in as kubeadmin user and run the following command:

After reading this, my reaction is to wonder why the documentation told me to log in as a developer if the only example it gives requires to be an admin.

**Comment by cfergeau** [comment]
`oc get pods` is empty right after cluster creation, so not that useful. `oc new-project` is also not really useful. Or do one of these, and point the user at more step by step documentation for their first openshift project?

**Comment by cfergeau** [comment]
This issue is still present.

**Comment by robin-owen** [comment]
Right, we just changed that command slightly, but never updated the docs to use a command accessible to the `developer` user.

Is there a good command that we can use here simply to verify that `oc` is interacting correctly with the cluster that doesn't require any additional privileges? I seem to vaguely recall a conversation about this, but I can't remember if we ever came up with a good answer.
